src/ext.js

```diff
@@ -272,6 +272,71 @@ export default function ext({ translator }) {
                     },
                   },
                 },
+                fontStyling: {
+                  component: 'item-selection-list',
+                  type: 'string',
+                  ref: 'style.textStyle',
+                  translation: 'Text Style',
+                  horizontal: true,
+                  multipleSelect: true,
+                  items: [
+                    {
+                      component: 'icon-item',
+                      icon: 'bold',
+                      value: 'bold',
+                      translation: 'Bold',
+                      labelPlacement: 'bottom',
+                    },
+                    {
+                      component: 'icon-item',
+                      icon: 'italic',
+                      value: 'italic',
+                      translation: 'Italic',
+                      labelPlacement: 'bottom',
+                    },
+                    {
+                      component: 'icon-item',
+                      icon: 'underline',
+                      value: 'underline',
+                      translation: 'Underline',
+                      labelPlacement: 'bottom',
+                    },
+                  ],
+                  defaultValue: {
+                    bold: true,
+                  },
```

Comment:
How is this stored? I assume every value has its own boolean? We should have these defaults in object.properties

Reply:
Yes, when one of them are toggled, they are added. Could add italic and underline as false as well